cmake/modules/Builddpdk.cmake

```diff
@@ -0,0 +1,27 @@
+if(WITH_SPDK)
+  if (CMAKE_SYSTEM_NAME MATCHES "Linux")
+    set(dpdk_env  "x86_64-native-linuxapp-gcc")
+    MESSAGE(STATUS "current platform: Linux ")
```

Comment:
drop this `MESSAGE()` please, if it only serves for the purpose of debugging.

Reply:
if clang is used, shall we have x86_64-native-linuxapp-**gcc** as well?

Reply:
I am not familiar with icc much. @optimistyzy what is your opinion?

Reply:
after talking with spdk team, icc is not supported by spdk until now.

Reply:
please see http://dpdk.org/doc/guides/linux_gsg/build_dpdk.html.

Reply:
yes, dpdk supports icc already, but for spdk, icc support is not trusted.